pkg/sentry/devices/nvproxy/version.go

```diff
@@ -292,6 +294,7 @@ func Init() {
 					nvgpu.KEPLER_CHANNEL_GROUP_A:  rmAllocSimple[nvgpu.NV_CHANNEL_GROUP_ALLOCATION_PARAMETERS],
 					nvgpu.TURING_CHANNEL_GPFIFO_A: rmAllocSimple[nvgpu.NV_CHANNEL_ALLOC_PARAMS],
 					nvgpu.AMPERE_CHANNEL_GPFIFO_A: rmAllocSimple[nvgpu.NV_CHANNEL_ALLOC_PARAMS],
+					nvgpu.HOPPER_CHANNEL_GPFIFO_A: rmAllocSimple[nvgpu.NV_CHANNEL_ALLOC_PARAMS],
```

Comment:
Also, what driver are you testing this on? Since 535.86.05, the definition of NV_CHANNEL_ALLOC_PARAMS has changed. I have sent #10001. Could you patch that, then update the handler for `HOPPER_CHANNEL_GPFIFO_A` on 535.86.05 and test on a driver which is 535.86.05+.

Reply:
The whole patch is developed on 535.104.05. We added the missing properties to the allocation class and I forgot to add them to the PR. I will cherry-pick your commit.